internal/events/blockchain_event_test.go

```diff
@@ -151,6 +151,7 @@ func TestContractEventWrongNS(t *testing.T) {
 
 }
 
+// TODO: Add test case for event not existing
```

Comment:
Are you going to address this todo?

Reply:
I believe this requires some more work with setting up mocks, etc. I can work on it is a separate PR.

Reply:
Code Coverage will force you to implement this test as you have added new code to return not created